talkgenerator/sources/unsplash.py

```diff
@@ -28,9 +29,15 @@ def get_unsplash_session():
 
 
 def search_photos_return_urls(query):
-    results = pu.search(type_="photos", query=query)
-    image_urls = []
-    for photo in results.entries:
-        link_download = photo.link_download
-        image_urls.append(link_download)
-    return image_urls
+    if pu:
```

Comment:
Not the most clear. What is `pu`

Reply:
Good call! It was something that was already named like that in the unsplash caller. Renamed it to "unsplash_session", as that indeed improves readability.